test/features/comments.feature

```diff
@@ -0,0 +1,76 @@
+@smoke
+Feature: Comments
+
+    Scenario: DQL-74 BDD-1a Dataset comment submission access (non-logged in user)
```

Comment:
Please remove all the `DQL-*` prefixes- they do not make sense in the open source project. These are internal references.

Besides, these will be outdated after a1 month.